Makefile

```diff
@@ -52,7 +52,7 @@ $(OUT)/cpu_instrs.h: tests/cpu_instrs.gb tests/rom2h.c
 
 $(OUT)/bench: $(OUT)/cpu_instrs.h prof.h cpu.c bench.c gameboy.h
 	$(VECHO) "  CC+LD\t$@\n"
-	$(Q)$(CC) -DENABLE_LCD=0 -o $@ cpu.c bench.c
+	$(Q)$(CC) -DENABLE_LCD=0 -DDEBUG -o $@ gbit/lib/*.c cpu.c bench.c 
```

Comment:
Avoid using `*`. Instead, specify the exact file names.

Reply:
Why did you enable `-DDEBUG` flags?

Reply:
I use `-DDEBUG` flags to change `__gb_read` `__gb_write` behavior, so that i can transfer data to `gbit` function

Reply:
The benchmark program should not depend on macro `DEBUG`.